README.md

```diff
@@ -18,7 +18,7 @@ to SMTP.
 
 Keila is ready for you to send newsletters with!
 
-You can easily deploy it using the official Docker image `pentacent/keila`.
+You can easily deploy it using the official Docker image `pentacent/keila` or use the [Docker Compose config](ops/docker-compose.yml) in this repo.
```

Comment:
I would suggest changing the wording. I see the docker-compose file as a way to quickly try Keila but not necessarily a proper deployment recommendation.

Reply:
Got it, will adjust the wording accordingly!